data/data/manifests/bootkube/openshift-kubevirt-infra-namespace.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-kubevirt-infra
```

Comment:
I'm not familiar with these infra namespaces. Are the infra namespaces for **all** platforms created in every cluster, regardless of the platform of the cluster? Or should only the infra namespace needed for the cluster be created?

Reply:
@staebler The namespaces creation originated from MCO, where each on-prem platform created its own  for workloads such as coredns, keepalived and so on. 
See https://github.com/openshift/machine-config-operator/pull/2098#discussion_r519750323

Reply:
today every cluster has all the empty namespaces with the names of all the platform exists.
even on GCP you will see namespaces for openstack, ovirt and all the others.
there is a proposal to change it in the future.
https://github.com/openshift/enhancements/pull/412